app/main/posts/common/post-actions.directive.js

```diff
@@ -51,9 +51,12 @@ function PostActionsDirective(
                     $location.path().indexOf('collections') === -1 &&
                     $location.path().indexOf('savedsearches') === -1) {
                     // Redirect to list
-                    $location.path('/views/data');
+                    $state.go('posts.data');
+                } else if ($state.current.name === 'posts.data' || $state.current.name === 'posts.data.detail') {
```

Comment:
This should really use `$state.includes('posts.data')`

Reply:
ah neat I didn't think of that.